src/utils/stateUpdateUtils.tsx

```diff
@@ -85,7 +86,7 @@ export const initializeData = async (dispatch: any, filters: Filters, exploreIsO
   const [records,
     reAggregatedRecords,
     estimateGradePrevalences] = await Promise
-      .all([api.getAirtableRecords(filters, exploreIsOpen),
+      .all([api.getAirtableRecords(filters, exploreIsOpen ? pins_columns : all_columns),

```

Comment:
Same with this one, `pins_modal_columns`

Reply:
On load I actually only want to be pulling in pins_columns. Only when we click on the modal do I want to be pulling in all the pins_modal_columns